device-trees/system-device-tree.dts

```diff
@@ -81,6 +81,63 @@
 		reg = <0xFF370000 0x1000>;
 	};
 
+        channel0vdev0vring0: channel0vdev0vring0@3ed40000 {
+                no-map;
+		#address-cells = <2>;
+		#size-cells = <2>;
+                reg = <0x3ed40000 0x4000>;
```

Comment:
Is there a mix of tabs and spaces ? or is the alignment just displaying strangely ?

Reply:
i apologize, it seems so. fixed in later commit.